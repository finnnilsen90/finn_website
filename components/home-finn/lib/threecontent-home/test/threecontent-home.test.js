const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Threecontent = require('../threecontent-home.js');

let newThreecontent = new Threecontent();

describe('component connection', () => {

    it('should be defined', () => {
        expect(Threecontent).toBeDefined();
    });


})  
    
describe('<Threecontent />', () => {

    const wrapper = render(<Threecontent />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})