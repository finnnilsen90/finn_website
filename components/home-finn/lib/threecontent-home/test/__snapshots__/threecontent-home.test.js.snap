// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Threecontent /> renders components 1`] = `
<div
  class="threecontent-home_Container font"
>
  <h1
    class="header"
  >
    Template
  </h1>
  <div
    class="test_content"
  >
    Test
  </div>
</div>
`;
